Source/hydro/Make.package

```diff
@@ -1,6 +1,9 @@
 # sources used with hydro
 
-CEXE_sources += Castro_hydro.cpp
+ifneq ($(USE_MHD),TRUE)
```

Comment:
Is there a specific conflict here?

Reply:
yes.  When we compile with USE_MHD=TRUE, the ctoprim call expects the B fields, but the hydro uses of it don't have B